shell/src/main/java/tachyon/shell/command/CountCommand.java

```diff
@@ -91,4 +91,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "count <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Displays the number of files and directories matching the specified prefix.";
```

Comment:
The class description isn't user-facing in the same way as the description text, so I'm fine with leaving updating it to another PR